obs/api/app/controllers/api/config.py

```diff
@@ -0,0 +1,35 @@
+from obs.libs import config
+from app.helpers.rest import response
+from flask import request, jsonify
+from flask_restful import Resource, reqparse
+
+def open_config():
+    data={}
```

Comment:
jangan lupa ditutup lagi file nya.
https://www.pythonforbeginners.com/cheatsheet/python-file-handling
coba pake with
https://stackoverflow.com/questions/9282967/how-to-open-a-file-using-the-open-with-statement

Reply:
file deleted cause we did't use config in local anymore